tgstation.dme

```diff
@@ -8816,6 +8816,7 @@
 #include "modular_skyrat\modules\xenos_skyrat_redo\code\xeno_types\sentinel.dm"
 #include "modular_skyrat\modules\xenos_skyrat_redo\code\xeno_types\spitter.dm"
 #include "modular_skyrat\modules\xenos_skyrat_redo\code\xeno_types\warrior.dm"
+#include "modular_skyrat\modules\clothing\suits\jacket.dm"
```

Comment:
After you make your edits make sure to open tgstation.dme in dreammaker so that it fixes the file sorting